apps/frontend/src/libs/components/page-layout/page-layout.tsx

```diff
@@ -13,7 +13,10 @@ const PageLayout = ({
 	children,
 	isLoading = false,
 }: Properties): JSX.Element => {
-	const { userPermissions } = useAppSelector(({ auth }) => auth);
+	const { projectUserPermissions, userPermissions } = useAppSelector(
+		({ auth }) => auth,
+	);
+	const projectPermissions = Object.values(projectUserPermissions).flat();
```

Comment:
Not sure how it works, don't we need to access project permission by project id. What if we have different permissions for different projects?

Reply:
On Page layout we just check permissions for sidebar. We don't have any projects here.
Maybe I don't get it right

Reply:
Got this, then this is ok